package-lock.json

```diff
@@ -1,17993 +1,8 @@
 {
   "name": "snow-to-flow",
   "version": "0.0.1",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
```

Comment:
Is this intentional? Curious about what it means

Reply:
...hmm. no idea